pkg/controllers/machine/disruption/emptiness.go

```diff
@@ -90,20 +90,20 @@ func (e *Emptiness) Reconcile(ctx context.Context, provisioner *v1alpha5.Provisi
 	}
 	// 5. If there are pods that are actively scheduled to the Node, remove the emptiness status condition
 	if len(pods) > 0 {
-		_ = machine.StatusConditions().ClearCondition(v1alpha5.MachineEmpty)
+		_ = nodeClaim.StatusConditions().ClearCondition(v1beta1.NodeEmpty)
 		if hasEmptyCondition {
-			logging.FromContext(ctx).Debugf("removing emptiness status condition from node")
+			logging.FromContext(ctx).Debugf("removing emptiness status condition since not empty")
```

Comment:
Maybe commas lead to more natural language than `since` everywhere
```suggestion
			logging.FromContext(ctx).Debugf("removing emptiness status condition, not empty")
```